content/en/synthetics/mobile_app_testing/_index.md

```diff
@@ -84,9 +83,23 @@ When configuring a new Mobile Application test, use snippets to automatically po
 
 ## Devices
 
-In the device selection screen, you can select mobile devices in either the EU or US locations. If a device is available in both locations, it automatically operates in each region.
+On the device selection screen, you can choose to test mobile devices that are located in either Europe (EU) or the United States (US). 
+To configure a mobile test to use either EU or US hosted devices:
 
-{{< img src="mobile_app_testing/mobile_device_locations.mp4" alt="Selecting one or more mobile devices in both the US and EU regions" video=true >}}
+1. Open the edit test page for an existing mobile test or create a new one.
+2. Expand the Devices section and choose the device location(s) for your test:
+   - **All locations**: Runs tests in both the EU and US.
+   - **EU**: Runs tests only in the EU.
+   - **US**: Runs tests only in the US.
+3. Use the Select Device(s) dropdown to select the devices you want to test on.
+4. Click **Save & Edit Recording** to save your test configuration.
+
+**Notes**: 
+- Tests executed on US-hosted devices are run from AWS US West (Oregon). Similarly, tests on EU-hosted devices are run from AWS Germany (Frankfurt).
+- Device locations cannot be combined. You must select either EU-hosted devices, US-hosted devices, or both, but they cannot be mixed within the same test configuration.
```

Comment:
Can you clarify? This says you can't combine, what do you mean by "or both" here?
`You must select either EU-hosted devices, US-hosted devices, **or both**, but they cannot be mixed within the same test configuration.`

Reply:
Is this what you mean?

```suggestion
- Device locations cannot be mixed within a single test configuration. Create separate configurations for EU-hosted and US-hosted devices if both are needed.
```

Reply:
Let me ask the PM about the wording here , this is confusing.